### PR TITLE
Generate class lists for model office

### PR DESCRIPTION
**Pull request by thomasleese**
This updates the scripts for the model office to generate class lists, and fixes a bug where importing the class list would remove the NHS numbers (and other attributes) of any patients that we already know about.

### CODE REVIEW

File: script/generate_model_office_data.rb [hunk 1]
```diff
@@ -241,11 +253,14 @@ def create_students_and_vaccinations_for(school:, team:, year_size_estimate:)
...
     vaccination_records += v
+
+    progress_bar.increment
   end
 
+  puts "Writing files"
+
   write_nominal_roll_to_file(students)
   write_vaccination_records_to_file(vaccination_records)
-  print "."
```
Comment (benilovj) [inline script/generate_model_office_data.rb:248]: aww my dots

**Comment by thomasleese** [comment]
> Not sure you meant to add [8ddb224](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/2118/commits/8ddb224b721d9105b68c4f9753f35426203323e0) onto this PR?

Probably should be two separate PRs, but I spotted this bug while testing class list imports... I can split it out if you'd prefer?

**Review by benilovj — APPROVED** [review]
👍 

* might be worth sticking the class import row changes on another PR
* If i'm reading the code correctly, the class lists assume nobody's moved at all. It would be good to introduce a small bit of variability into the class lists to better simulate reality (e.g. 5% of children have moved to a different school in the area, for instance, and another 5% have left the area altogether). But this could be a follow-up PR.

**Comment by thomasleese** [comment]
> If i'm reading the code correctly, the class lists assume nobody's moved at all. It would be good to introduce a small bit of variability into the class lists to better simulate reality (e.g. 5% of children have moved to a different school in the area, for instance, and another 5% have left the area altogether). But this could be a follow-up PR.

Yep definitely agree.

**Pull request merged**